api/paidAction/itemCreate.js

```diff
@@ -11,9 +11,34 @@ export const paymentMethods = [
   PAID_ACTION_PAYMENT_METHODS.PESSIMISTIC
 ]
 
+export const BIO_COST = 1000n
+
+export async function findRootItem (models, parentId) {
+  const item = await models.item.findFirst({
+    where: { id: Number(parentId) },
+    select: { rootId: true }
+  })
+
+  return models.item.findFirst({
+    where: { id: Number(item.rootId || parentId) },
+    include: { sub: true }
+  })
+}
+
+export async function getBaseCost ({ models, bio, parentId, subName }) {
+  if (bio) return BIO_COST
+
+  if (parentId) {
+    const rootItem = await findRootItem(models, parentId)
+    return rootItem.bio ? BIO_COST : satsToMsats(rootItem.sub?.replyCost)
```

Comment:
I noticed you need the same check as in components/reply.js:177 here for saloon replies else this will return `null` because `rootItem.sub` is not defined and then `itemCreate` will throw `Cannot convert null to a BigInt`.

```suggestion
    if (rootItem.bio || !rootItem.sub) return BIO_COST
    return satsToMsats(rootItem.sub.replyCost)
```

I would then also rename `BIO_COST` to something else like `DEFAULT_COST` (maybe you have a better idea for name).

Reply:
After applying today's patch I moved on and missed it 🫠, sorry and thanks ek!